cmake/Modules/FindwxWidgetsTB.cmake

```diff
@@ -848,6 +848,27 @@ else()
   endif()
 endif()
 
+if(wxWidgets_FOUND)
+  find_file(_filename wx/version.h PATHS ${wxWidgets_INCLUDE_DIRS} NO_DEFAULT_PATH)

```

Comment:
`_filename` is not very descriptive, maybe rename it to `version_file` or whatever.